template.yaml

```diff
@@ -12,7 +12,7 @@ Metadata:
     Author: Unit
     SpdxLicenseId: MIT
     LicenseUrl: LICENSE
-    Labels: ['${CODEBUILD_RESOLVED_SOURCE_VERSION}', '${GIT_REPO}']
+    Labels: ['git-repo:${GIT_REPO}', 'git-id:${CODEBUILD_RESOLVED_SOURCE_VERSION}', 'build@${BUILD_TIMESTAMP}']
```

Comment:
CodeBuild provides CODEBUILD_START_TIME but you probably wanted exact time of build?

Reply:
Using compact time format without spaces